app/controllers/staff_info_controller.rb

```diff
@@ -0,0 +1,80 @@
+class StaffInfoController < ApplicationController
+  before_action :authorize_prison_request
+
+  STAFF_INFO_ENDPOINT = ENV.fetch('STAFF_INFO_ENDPOINT', nil)
```

Comment:
This is inconsistent with the way configuration is fetched from ENV elsewhere. Can it be in `config/application.rb`?